ci/hera_qm_tests.yml

```diff
@@ -0,0 +1,21 @@
+name: hera_qm_tests
+channels:
+  - defaults
+  - conda-forge
+dependencies:
+  - numpy>=1.10
+  - pip
+  - astropy>=2.0.16
```

Comment:
Since you're only testing in python 3 you might want to change this to be `astropy>=3.2.3`, that was when the UNSO changes were.

Reply:
Will do.